docs/plugins/composable_resources.md

```diff
@@ -276,6 +276,29 @@ acPostProcForDataCopyReceived(*leaf_resource) {
 }
 ```
 
+Detached mode was added as a configuration option for UnixFileSystem resources in release 4.3.1.  When detached mode is enabled, any server may serve up requests for the resource as long as all servers share a common mounted vault.
+
+To enable detached mode, the host_mode setting in the context string should be set to "detached" as in the following example:
+
+```
+irods@hostname:~/ $ iadmin mkresc detached_resc3 unixfilesystem hostname.example.org:/common/mount/point "host_mode=detached"
+```
+
+If an administrator wants to restrict the number of hosts that may serve the request, a host_list parameter with a comma separated list of hosts may be added to the context string.
+
+```
+irods@hostname:~/ $ iadmin mkresc detached_resc3 unixfilesystem hostname.example.org:/common/mount/point "host_mode=detached;host_list=host2.example.org,host3.example.org"
+```
+
+The host assigned to the host field for the resource is assumed to be able to serve up any request, so in the above case either hostname, host2, or host3 may serve up the request.
+
+The following are some rules around using detached mode:
+
+1. If host_list does not exist but host_mode=detached, all hosts are assumed to be able to service requests for the resource.
```

Comment:
`but host_mode=detached,` -> `but host_mode=detached does,`?

Is this statement true if host_list is empty (i.e. `host_mode=detached;host_list=`)?

Reply:
It will behave as if detached mode is disabled.  The host in the resource configuration will essentially be the only host in the host list.